src/codeu/chat/common/BasicView.java

```diff
@@ -24,32 +24,37 @@
 
 // BASIC VIEW
 //
-//   The view component in the Model-View-Controller pattern. This component
-//   is used to read information from the model where the model is the current
-//   state of the server. Data returned from the view should be treated as
-//   read only data as manipulating any data returned from the view may
-//   have no effect on the server's state.
+// The view component in the Model-View-Controller pattern. This component
+// is used to read information from the model where the model is the current
+// state of the server. Data returned from the view should be treated as
+// read only data as manipulating any data returned from the view may
+// have no effect on the server's state.
 
 public interface BasicView {
 
   // GET USERS
   //
-  //   Return all users whose id is found in the given collection.
+  // Return all users whose id is found in the given collection.
   Collection<User> getUsers();
 
   // GET ALL CONVERSATIONS
   //
-  //   Return a summary of each converation.
+  // Return a summary of each converation.
   Collection<ConversationHeader> getConversations();
 
   // GET CONVERSATIONS
   //
-  //   Return all conversations whose id is found in the given collection.
+  // Return all conversations whose id is found in the given collection.
   Collection<ConversationPayload> getConversationPayloads(Collection<Uuid> ids);
 
   // GET MESSAGES
   //
-  //   Return all messages whose id is found in the given collection.
+  // Return all messages whose id is found in the given collection.
   Collection<Message> getMessages(Collection<Uuid> ids);
 
+  // GET SERVER INFO
+  //
+  // Return server info
```

Comment:
it's usually nice to explain that it can return null in case of an error (although i see that the other method docs aren't nice like that)